api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function login(email, password) {
+  try {
+    const response = await axios.post('/api/login', { email, password });
+    return response.data.token;
+  } catch (error) {
```

Comment:
Error handling in the login function is not robust. The function should return a specific error message or code, rather than just logging the error to the console.